### ISSUE TITLE
Possible memory leak fix (as a response to the v4.1 tweet)

### ISSUE THREAD

**Issue by Burgerballs** [Clowfoe/IMPOSTOR-UPDATE#31]
Through MainMenuState i can tell that you guys are using 0.4.0, so the way to help stop the lua memory leaks you mentioned is by checking out the 0.4.2 Psych commit, which made drastic improvements to the lua, where ShadowMario claimed to fix a lot of memory leaks on the way.

https://github.com/ShadowMario/FNF-PsychEngine/commit/cae36fa1821aa201236995bd03919441e1ef069b#diff-729dcc53e1e68078d390c997299f9da44feaa46f1d9b8f53b8753b9df91c320bL685

**Comment by CoolingTool** [comment]
imagine

**Comment by Burgerballs** [comment]
> it would be funny if this mods psych engine gets updated to the latest ver

Nah, if i were them i'd just add the fixes and no update thingies

**Comment by LeanDapper** [comment]
> Through MainMenuState i can tell that you guys are using **0.4.0**, so the main way to help stop the lua memory leaks you mentioned is by checking out the **0.4.2** Psych commit, which made drastic improvements to the lua, where ShadowMario claimed to fix a lot of memory leaks on the way.
> 
> [ShadowMario/FNF-PsychEngine@cae36fa#diff-729dcc53e1e68078d390c997299f9da44feaa46f1d9b8f53b8753b9df91c320bL685](https://github.com/ShadowMario/FNF-PsychEngine/commit/cae36fa1821aa201236995bd03919441e1ef069b#diff-729dcc53e1e68078d390c997299f9da44feaa46f1d9b8f53b8753b9df91c320bL685)

I'm checking this out
